projects/rocprofiler-systems/docs/how-to/understanding-rocprof-sys-output.rst

```diff
@@ -358,7 +319,58 @@ set ``ROCPROFSYS_OUTPUT_PREFIX="%argt%-"``, and let ROCm Systems Profiler cleanl
    an ``%arg0%`` of ``/usr/bin/foo`` translates to ``usr_bin_foo``. Additionally, any ``%arg<N>%`` keys which
    do not have a command line argument at position ``<N>`` are ignored.
 
-Perfetto output
+.. _rocprof_sys_rocpd_output:
+
+ROCm Profiling Data (rocpd) output
+=========================================
+
+Use the ``ROCPROFSYS_USE_ROCPD`` setting to trigger the ROCm Systems Profiler to output a
+SQLite3 database. The ROCm Profiling Data (or ``rocpd``) database will soon be the default output
+format. To output in `rocpd` format, ROCProfiler-SDK version 1.0.0 or later is required (introduced in ROCm 7.0.0).
+
+Features of rocpd format
+-----------------------------------------------
+
+* **Comprehensive Data Model**: Consolidates all profiling artifacts including
+execution traces, performance counters, hardware metrics, and contextual metadata
+within a single SQLite3 database file (`.db` extension).
+
+* **Standards-Compliant Access**: Supports querying through industry-standard SQL
+interfaces including command-line tools (``sqlite3`` CLI), programming language
+bindings (Python ``sqlite3`` module, C/C++ SQLite API), and database management
+applications.
+
+* **Advanced Analytics Integration**: Facilitates sophisticated post-processing
+workflows through custom analytical scripts, automated reporting systems, and
+integration with third-party visualization and analysis frameworks that provide
+SQLite3 connectivity.
+
+Generating rocpd Output
++++++++++++++++++++++++
+
+To generate profiling data in the rocpd format, add ``ROCPROFSYS_USE_ROCPD=ON`` to your profiling configuration.
+
+.. code-block:: shell
+
+   export ROCPROFSYS_USE_ROCPD=ON
+   export ROCPROFSYS_USE_TRACE=OFF # disabling default Perfetto output
+   rocprof-sys-sample -- ./your_application
+
+See :doc:`configuring runtime options <./configuring-runtime-options>` for additional
```

Comment:
The example above makes it look like this would work only with `rocprof-sys-sample` or when collecting traces in sampling mode. If that is not true and it would work with any type of rocprof-sys profile collection, could you elaborate that here?